### PR TITLE
Docs 11129 mule onboarding test2 ls

### PR DESCRIPTION
**Pull request by luanamulesoft**
Pull request test

### CODE REVIEW

File: vm/2.0/modules/ROOT/pages/vm-examples.adoc [hunk 1]
```diff
@@ -14,6 +14,9 @@ Configure the *Publish* operation and *Listener* source to publish a message tha
 * xref:vm-publish-response.adoc[Publish a Message and Get a Response] +
 Configure the *Publish consume* operation to publish a message and get a response.
 
+New Branch Test
```
Comment (kevintroller) [inline vm/2.0/modules/ROOT/pages/vm-examples.adoc]:
```suggestion
This is a suggestion text.
```

File: vm/2.0/modules/ROOT/pages/vm-examples.adoc [hunk 2]
```diff
@@ -14,6 +14,9 @@ Configure the *Publish* operation and *Listener* source to publish a message tha
...
+Testing new line
```
Comment (luanamulesoft) [inline vm/2.0/modules/ROOT/pages/vm-examples.adoc]:
```suggestion
Testing new comment
```

File: vm/2.0/modules/ROOT/pages/vm-examples.adoc [hunk 3]
```diff
@@ -14,6 +14,9 @@ Configure the *Publish* operation and *Listener* source to publish a message tha
...
+
```
Comment (luanamulesoft) [inline vm/2.0/modules/ROOT/pages/vm-examples.adoc]: I ould include another line

File: vm/2.0/modules/ROOT/pages/vm-examples.adoc (hunk 2 above)
Comment (kevintroller) [inline vm/2.0/modules/ROOT/pages/vm-examples.adoc]:
```suggestion
Testing new line, do not accept me!
```

File: vm/2.0/modules/ROOT/pages/vm-examples.adoc [hunk 4]
```diff
@@ -14,6 +14,9 @@ Configure the *Publish* operation and *Listener* source to publish a message tha
 * xref:vm-publish-response.adoc[Publish a Message and Get a Response] +
 Configure the *Publish consume* operation to publish a message and get a response.
 
+This is a suggestion text.
+Testing new line. Please review.
```
Comment (kevintroller) [inline vm/2.0/modules/ROOT/pages/vm-examples.adoc]:
```suggestion
Delete me!
```

**Comment by kevintroller** [comment]
Closing MuleSoft Onboarding TEST PR

**Pull request closed without merging**